trex.org

```diff
@@ -252,138 +252,82 @@
+  by a factor of $r^{-\ell_s}$.
+
+  The radial function is a linear combination of
+  $N_{\text{prim}}$ *primitive* functions. These primitives may be
+  of Slater-type or Gaussian-type, and are parameterized by
+  exponents $\gamma_{ks}$ and contraction coefficients $a_{ks}$.
+
+  Slater orbitals are written as:
+
+  \[
+  R_s^{\text{STO}}(\mathbf{r}) = \mathcal{N}_s  \left| \mathbf{r} - \mathbf{R}_A \right|^{n_s-1-\ell_s}
+  sum_{k=1}^{N_{\text{prim}}}
+  _{ks}\,a_{ks}\,
+  exp\left( -\zeta_{ks} \left| \mathbf{r} - \mathbf{R}_A \right| \right)
+  \]
+
+  Gaussian-type orbitals (GTOs) are usually defined as:
+
+  \[
+  R_s^{\text{GTO}}(\mathbf{r}) = \mathcal{N}_s \left| \mathbf{r} - \mathbf{R}_A \right|^{n_s}
+  \sum_{k=1}^{N_{\text{prim}}} f_{ks} \, a_{ks}\,
+  \exp\left( -\gamma_{ks} \left| \mathbf{r} - \mathbf{R}_A \right|^2 \right)
+  \]
+
+  GTO-based codes usually use *spherically pure* Gaussians, in which
+  case the radial prefactor exponent is zero, as the
+  $r^{\ell_s}$ factor is already included in the solid harmonics.
+  TREXIO is flexible enough to allow the implementation of Cartesian
+  Gaussians in terms of spherical shells. For instance, a Cartesian
+  d shell consists of 6 functions, which are linear combinations of
+  the 5 spherical d functions with $n_s=0$ and a spherical s
+  function with $n_s=2$. More generally, a Cartesian shell of
+  Cartesian angular momentum $\lambda_s$ will give rise to a
+  spherical shell with $\ell_s = \lambda_s$ ($n_s=0$), $\ell_s =
+  \lambda_s -2$ ($n_s=2$), $\ell_s = \lambda_s -4$ ($n_s=4$), etc.
+
+  Standard basis sets assume that the primitive functions are
+  normalized. In that case, a normalization constant $f_{ks}$ must
+  be computed and applied to each primitive. This normalization
+  constant usually depends on the exponent and on the quantum
+  numbers $n$ and $\ell$. If this normalization is not required,
+  then $f_{ks} = 1$.
+
+  Similarly, some codes expect the contracted basis functions
+  $R_s(\mathbf{r})$ to be normalized. In this case, an additional
+  normalization factor $\mathcal{N}_s$ must be computed. If the
+  full basis function is not normalized, then $\mathcal{N}_s = 1$.
 
 *** Numerical orbitals
 
     TREXIO supports numerical atom centered orbitals. The implementation is
     based on the approach of FHI-aims [Blum, V. et al; Ab initio molecular
     simulations with numeric atom-centered orbitals; Computer Physics
-    Communications 2009]. These orbitals are
-    defined by the atom they are centered on, their angular momentum and a
-    radial function $R_s$, which is of the form
+    Communications 2009]. These orbitals are defined by the atom they
+    are centered on, their angular momentum and a radial function
+    $R_s$, which is of the form
     \[
-    R_s(\mathbf{r}) = \mathcal{N}_s \frac{u_i(\mathbf{r})}{r^{-n_s}}.
+    R_s(\mathbf{r}) = \mathcal{N}_s \frac{u_i(\mathbf{r})}{r^{n_s}}.
```

Comment:
I wonder about this expression since usually they define $R_s(r) = u_s(r)/r$ but here one also has the additional $r^{-\ell_s}$ term.

Reply:
Yes. I suppose the one who introduced this section was also confused with the original text of the GTO/STO... I am not familiar with this, I need to check.

Reply:
did you get anywhere with this